ReactiveXComponent/Configuration/XCConfiguration.cs

```diff
@@ -98,15 +98,15 @@ public string GetPublisherTopic(string component, string stateMachine, int event
             }   
         }
 
-        public string GetConsumerTopic(string component, string stateMachine, int eventCode)
+        public string GetSubscriberTopic(string component, string stateMachine)
         {
             try
             {
-                return _parser.GetConsumerTopic(component, stateMachine);
+                return _parser.GetSubscriberTopic(component, stateMachine);
             }
             catch (NullReferenceException ex)
```

Comment:
Weird !!
do not catch or throw NullReferenceException !!
Fix the cause of your nullReference